lib/nimble_template/helpers/dependency.ex

```diff
@@ -1,5 +1,34 @@
 defmodule NimbleTemplate.DependencyHelper do
-  def fetch_and_install_dependencies(),
+  alias NimbleTemplate.Generator
+
+  def fetch_and_install_dependencies,
     do:
       if(Mix.shell().yes?("\nFetch and install dependencies?"), do: Mix.shell().cmd("mix deps.get"))
+
+  def order_dependencies! do
+    file_content = File.read!("mix.exs")
+
+    dependencies = extract_dependencies(file_content)
+
+    ordered_dependencies =
+      dependencies
+      |> String.split(",\n")
+      |> Enum.sort()
+      |> Enum.join(",\n")
+
+    Generator.replace_content("mix.exs", dependencies, ordered_dependencies)
+  end
+
+  def extract_dependencies(contents) do
+    [_, deps_with_tail] =
```

Comment:
In terms of naming, while `tail` sounds idiomatic, `deps_with_tail` is not that descriptive as it requires reading the whole code to understand that the tail means the `end` block for the method.

Reply:
Updated the name and code formatting. So more variants of the test are passing now ✅

I think we should open another PR to fix the failing test variants